### PR TITLE
docs for webview2 installer preview

### PR DESCRIPTION
**Pull request by liminzhu**
**DO NOT MERGE**

Doc update for WebView Runtime and installer preview happening soon. I will follow up on the Insert sample code or step-by-step guides in a later commit. Target date for merging 6/23.

### CODE REVIEW

**Comment by jm-trd-ms** [comment]
Converted to draft per "DO NOT MERGE" note.

**Comment by jm-trd-ms** [comment]
Added editorial review.

**Comment by liminzhu** [comment]
@jm-trd-ms I'm seeing a lot of edits unrelated to this installer preview/PR (the changes in release notes, versioning, and roadmap). Can you move them to a separate PR?

**Comment by jm-trd-ms** [comment]
Quick fix to a link in release notes.

**Comment by jm-trd-ms** [comment]
Cleaned up merge conflict with origin.

File: microsoft-edge/webview2/concepts/distribution.md [hunk 1]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+The evergreen distribution mode ensures that your app is taking advantage of the latest features and security updates.  The evergreen distribution mode has the following characteristics.  
 
-There will be support for two distribution options using the WebView2 Runtime: evergreen, and fixed version.
+*   The web platform updates automatically without additional effort from you.  
+*   All applications that leverage the evergreen distribution mode use a shared copy of the platform binaries, which saves disk space.  
 
-Evergreen will be the recommended distribution model for most developers. In this mode, updates are pushed automatically to the WebView2 Runtime without additional effort from your application. The evergreen model ensures that your app is taking advantage of the latest features and security updates for hosted web content.
+You may choose from multiple channels WebView2 for applications to use as the web platform.  By default, WebView2 targets the most stable channel available on the device that meets the minimum version requirement of the WebView2 SDK.  The following channels are listed in order from most to least stable channel.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:24]:
```suggestion
There are multiple channels WebView2 applications may use as the web platform.  By default, WebView2 targets the most stable channel available on the device that meets the minimum version requirement of the WebView2 SDK.  The following channels are listed in order from most to least stable channel.  
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 2]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+> [!NOTE]
+> The evergreen distribution model is recommended for most developers.  
 
-The Microsoft Edge WebView2 Runtime will package the browser binaries optimized for use by WebView2 applications. It will function standalone or side-by-side with a client Edge Browser installed. Installing the runtime once will support many WebView2 applications running on the client machine. Installing the runtime will not appear as a browser to end-users, and will have no desktop shortcuts, start menu entry point, or protocol registration.
+Microsoft Edge Stable Channel runtime is not a valid target for WebView2, and the reasons are described later.  
 
-Applications that use the WebView2 Runtime will need to ensure that the runtime installation completed. To ensure your application installs the runtime as a dependency, you’ll be able to add the runtime to your install flow. 
+For more information about versioning, see [Versioning][ConceptsVersioning].  
+
+For more information about global methods, see [Globals][ReferenceWin3209538WebviewIdl].
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:38]:
```suggestion
For more information about versioning, see [Versioning][ConceptsVersioning] and [Globals][ReferenceWin3209538WebviewIdl].  
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 3]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
 
-The Microsoft Edge WebView2 Runtime will package the browser binaries optimized for use by WebView2 applications. It will function standalone or side-by-side with a client Edge Browser installed. Installing the runtime once will support many WebView2 applications running on the client machine. Installing the runtime will not appear as a browser to end-users, and will have no desktop shortcuts, start menu entry point, or protocol registration.
+Microsoft Edge Stable Channel runtime is not a valid target for WebView2, and the reasons are described later.  
 
-Applications that use the WebView2 Runtime will need to ensure that the runtime installation completed. To ensure your application installs the runtime as a dependency, you’ll be able to add the runtime to your install flow. 
+For more information about versioning, see [Versioning][ConceptsVersioning].  
+
+For more information about global methods, see [Globals][ReferenceWin3209538WebviewIdl].  
+
+### Understand the WebView2 Runtime and installer
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:40]:
```suggestion
### Understand the WebView2 Runtime and installer (Preview)  
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 4]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
 
-Applications that use the WebView2 Runtime will need to ensure that the runtime installation completed. To ensure your application installs the runtime as a dependency, you’ll be able to add the runtime to your install flow. 
+For more information about versioning, see [Versioning][ConceptsVersioning].  
+
+For more information about global methods, see [Globals][ReferenceWin3209538WebviewIdl].  
+
+### Understand the WebView2 Runtime and installer  
+
+> [!NOTE]
+> The following information supports the WebView2 Runtime \(Preview\).
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:43]:
```suggestion

```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 5]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+
+Today during the preview, the Evergreen WebView2 Runtime and Microsoft Edge Dev Channel are updated at the same time and have the same build.  In the future during the Preview, the WebView2 Team plans to update the WebView2 Runtime and match the same build as the Microsoft Edge Beta Channel.  In the future when WebView2 reaches General Availability \(GA\), the WebView2 Team plans to update the WebView2 Runtime and match the same build as the Microsoft Edge Stable Channel.  After GA, applications should use the WebView2 Runtime in production.  
+
+> [!IMPORTANT]
+> Do not ship WebView2 applications in production during the preview.  
+
+The WebView team recommends the following workflow to ensure the Evergreen WebView2 Runtime is available.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:52]:
```suggestion
We recommend the following workflow to ensure the Evergreen WebView2 Runtime is available.  
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 6]
```diff
@@ -3,29 +3,86 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+Consider the following recommendations during the preview.  
+
+1.  Make sure to use the [Evergreen WebView2 Runtime and Installer ][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application must include the installer.  
+1.  For developing your application, you may use the Evergreen WebView2 Runtime.  However, as the runtime shifts from the Dev Channel to the Beta Channel or Stable Channel, the runtime build number may not meet the most recent preview WebView2 SDK minimum version requirements.  If you wish to use the most recent SDK, install the Microsoft Edge Canary channel to ensure a compatible build is available on the device.  For more information about versioning, see [Versioning][ConceptsVersioning].  
+1.  To test your web content for compatibility with changes to the platform not available in the Stable Channel, use the appropriate non-Stable Channel as required.  
+
+## Fixed version distribution mode  
+
+> [!NOTE]
+> The following information supports the fixed version distribution model.  The fixed version distribution model is currently not available.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/distribution.md:74]:
```suggestion
> [!NOTE]
> The fixed version distribution model is currently not available.  
```

File: microsoft-edge/webview2/concepts/distribution.md [hunk 7]
```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+# Distribution of applications using WebView2  
 
-The WebView2 control utilizes the Microsoft Edge \(Chromium\) browser. When distributing your app, ensure that the Edge browser is installed on all user machines where your application runs. The WebView2 control uses the most stable version of the browser that’s installed on a machine. For example, you may have Stable, Beta, Dev, and Canary installed at the same time, and in this situation, WebView2 controls run in the Stable channel. Ensure you review the release notes to ensure that the version of the browser installed on your machines that run your WebView2 controls meet the minimum version requirements.
+The WebView2 control utilizes the Microsoft Edge \(Chromium\) platform.  When packaging and distributing your app, make sure a copy of the platform or the WebView2 Runtime is present before the app starts.  The following page describes how you \(the developer\)are able to ensure that the WebView2 Runtime is installed and use of the two distribution modes for your WebView2 application:  [Evergreen](#evergreen-distribution-mode) and [Fixed version](#fixed-version-distribution-mode).
```
Comment (liminzhu) [inline microsoft-edge/webview2/concepts/distribution.md:15]: space after `(the developer)`

File: microsoft-edge/webview2/concepts/distribution.md [hunk 8]
```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+1.  Microsoft Edge Beta Channel  
+1.  Microsoft Edge Dev Channel  
+1.  Microsoft Edge Canary Channel    
 
-### Microsoft Edge WebView2 Runtime
+> [!NOTE]
+> The evergreen distribution model is recommended for most developers.  
 
-The Microsoft Edge WebView2 Runtime will package the browser binaries optimized for use by WebView2 applications. It will function standalone or side-by-side with a client Edge Browser installed. Installing the runtime once will support many WebView2 applications running on the client machine. Installing the runtime will not appear as a browser to end-users, and will have no desktop shortcuts, start menu entry point, or protocol registration.
+Microsoft Edge Stable Channel runtime is not a valid target for WebView2, and the reasons are described later.
```
Comment (liminzhu) [inline microsoft-edge/webview2/concepts/distribution.md:34]: Mark as IMPORTANT. Also, delete the word runtime after `Microsoft Edge Stable Channel`

File: microsoft-edge/webview2/concepts/distribution.md [hunk 9]
```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+Depending on your scenario, you may need to change the above workflow.  For example, your application installer may download the Evergreen WebView2 Runtime Installer instead of including it in your application package.  
+
+> [!NOTE]
+> Both the WebView2 Runtime and WebView2 Runtime installer are in preview.  The preview has a limited initial scope and is only available as a standalone, per-machine install on Windows 10 on x64.  In the future, support for Windows 7, x86, and ARM64 is planned.  
+
+### Best practices for using WebView2 Runtime and non-stable Microsoft Edge channels  
+
+Consider the following recommendations during the preview.  
+
+1.  Make sure to use the [Evergreen WebView2 Runtime and Installer ][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application must include the installer.
```
Comment (liminzhu) [inline microsoft-edge/webview2/concepts/distribution.md:63]: `Must` is a strong word and not exactly true. I'd say is advised or should.

File: microsoft-edge/webview2/concepts/distribution.md [hunk 10]
```diff
@@ -3,29 +3,82 @@ description: Distribution options when releasing an app using Microsoft Edge Web
...
+1.  To test your web content for compatibility with changes to the platform not available in the Stable Channel, use the appropriate non-Stable Channel as required.  
+
+## Fixed version distribution mode  
+
+> [!NOTE]
+> The fixed version distribution model is currently not available.  
+
+For constrained environments, the WebView team plans to support a fixed version \(previously called bring-your-own\) distribution model.  The fixed version distribution model has the following benefits and limitations.  
+
+*   Allows you to select and package a specific version of the WebView2 Runtime in your application installer.
```
Comment (liminzhu) [inline microsoft-edge/webview2/concepts/distribution.md:74]: Can we use original wording in the doc? There are a lot of overlap in between bullets and this is not how I would describe benefits/limitations.

File: microsoft-edge/webview2/concepts/distribution.md (hunk 7 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/distribution.md:15]: Updated.

File: microsoft-edge/webview2/concepts/distribution.md (hunk 8 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/distribution.md:34]: Updated.
```md
> [!IMPORTANT]
> Microsoft Edge Stable Channel is not a valid target for WebView2, and the reasons are described later.  
```

File: microsoft-edge/webview2/concepts/distribution.md (hunk 9 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/distribution.md:63]: Updated.  
```md
1.  Make sure to use the [Evergreen WebView2 Runtime and Installer][Webview2Installer] to develop or test your packaging and distribution pipeline.  In the future, your production application should include the installer.  
```

File: microsoft-edge/webview2/concepts/distribution.md (hunk 10 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/distribution.md:74]: Updated.
```md
## Fixed version distribution mode  

> [!NOTE]
> The fixed version distribution model is currently not available.  

For constrained environments, there are plans to support a fixed version \(previously called bring-your-own\) distribution mode.  The fixed version distribution mode allows you to select and package a specific version of the WebView2 Runtime.  The fixed version distribution mode allows you to control which version of the WebView2 Runtime is used by your application, and when user machines are updated.  The fixed version distribution mode does not receive any automatic updates, and you should plan to manually apply updates.  
```

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**